saas/api/metrics.py

```diff
@@ -237,24 +237,26 @@ class RevenueMetricAPIView(BeforeMixin, ProviderMixin, APIView):
         }
     """
     def get(self, request, *args, **kwargs):
+        dates = convert_dates_to_utc(month_periods(13, self.ends_at, tz=self.timezone))
```

Comment:
With how `aggregate_transactions_change_by_period` was changed to use 13 date points / 12 periods (ie. removed `trail...`), it seems here a call to `month_periods(12,...` is in order.